### DISCUSSION TITLE
How get rid of "BlobBuilder" warnings?

### DISCUSSION THREAD

**Discussion by janober** [pyodide/pyodide#3538] [General]
Hi,

Currently, every time I execute some Python code, it displays two warnings:
```
warning: no blob constructor, cannot create blobs with mimetypes
warning: no BlobBuilder
```
How can I disable them? Or how can I add a BlobBuilder to make pyodide happy?

I looked through the docs, issues, and discussions but all mentions I could find of them is about things totally unrelated to them. Start to wonder if I am the only person who minds them.

Thanks!

**Comment by hoodmane** [comment]
Are you using node? Which version?

**Comment by janober** [comment]
Yes. Node.js version 16 and some users will also potentially use 14.

**Comment by hoodmane** [comment]
Blob was added in node v14.18 in the `node:buffer` module and stabilized in node v16.17. In v18.0, it was added to the node globals, so in versions >= 18 no action is needed. 

In node >= v14.18, you can do:
```js
globalThis.Blob = require("node:buffer").Blob;
```
before running `loadPyodide`.

If you are using a version between v14.8 and v16.17, it will still print a warning:
```
(node:11123) ExperimentalWarning: buffer.Blob is an experimental feature. This feature could change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
```
If you want to prevent this warning too, you should use cross-blob: https://www.npmjs.com/package/cross-blob.
We should probably at least add an FAQ entry about this...

**Reply by janober** [reply]
Great thank you very much. Will give it a try.
